lmfit/ui/ipy_fitter.py

```diff
@@ -116,12 +107,11 @@ def close(self):
         self.max_checkbox.close()
 
     def _repr_html_(self):
-        box = Box()
+        box = HBox()
         box.children = [self.value_text, self.vary_checkbox,
                         self.min_text, self.min_checkbox,
                         self.max_text, self.max_checkbox]
         display(box)
```

Comment:
Oops, sorry totally missed that.
I'll revert those changes, but I cant test the widgets then though...